examples/cifar_cnntransformer/configs/CF10-StandardCNN-huge.yaml

```diff
@@ -5,4 +5,5 @@ CNN:
   CONV_LAYERS: [[16,3], [32,3], [64,3], [32,1], [64,3], [128,3], [256,3], [256, 1]]
   OUTPUT_SHAPE: (-1, 256, 8, 8)
 
-OUTPUT_DIR: './outputs/StandardCNN-large'
+OUTPUT:
+  OUT_DIR: './outputs/StandardCNN-large'
```

Comment:
Double check please, the file name says huge but here it says large

Reply:
Change made.